macros/core_web_vital_page_groups.sql

```diff
@@ -0,0 +1,13 @@
+{% macro core_web_vital_page_groups() %}
+  {{ return(adapter.dispatch('core_web_vital_page_groups', 'snowplow_web')()) }}
+{%- endmacro -%}
+
+{% macro default__core_web_vital_page_groups() %}
+
+  case when page_url like '%/product%' then 'PDP'
+      when page_url like '%/list%' then 'PLP'
+      when page_url like '%/checkout%' then 'checkout'
+      when page_url like '%/home%' then 'homepage'
+      else 'other' end
+
+{% endmacro %}
```

Comment:
What's the docs plan for CWV outside of the accelerator? A tab under web like consent/conversions in the current docs PR? That where we tell them to overwrite this?

Reply:
Yes, I have added a separate tab. The general macros are specified where you started to do so and this one will sit inside that page.